docs/content/en/docs/concepts/architecture/components/KLTcomponents.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
We should name this file `_index.md`, or is there a particular reason for this `KLTcomponents.md` name?

Reply:
No there is no reason I thought to name the file according to their content😅should I change this to `_index.md`?

Reply:
so there are two ways of naming files in hugo - see https://gohugo.io/content-management/organization/

1. making a folder and using an `_index.md`
2. no folder and name it like the title or something similar like you did with `kltcomponents.md` 

currently we are mixing both of those, and we should use one of them. I am not sure if we do have a global rule, to always use directories (first approach) but imho it makes only sense, if you need to add images, or have subsections to this one.

As we do not have sub sections, and no assets or anything, you can also use the second approach - in my opinion. I am not sure what the rule for keptn is here. Maybe @StackScribe or @thisthat can shed some light :)

Reply:
For now, I renamed to `_index.md` file as all file follows this conevention.

Reply:
Also, need suggestion for the name here  @aepfli 
![image](https://user-images.githubusercontent.com/49230384/224909079-6ba4a497-5beb-454d-a8fb-46778d1f480a.png)

What will be better to put here?
- KLT components
- Keptn Lifecycle Components.

Reply:
I would skip the `KLT` and call it components. We're in the KLT documentation that naming is redundant.

Furthermore, `Working` is not a good link title - `How it works?` might be better, or maybe this content would be suitable for the `Architecture` page anyways.

Reply:
ok I am adding `How it works?` and `components`.

Reply:
![image](https://user-images.githubusercontent.com/49230384/224937427-65c9a788-b8bc-4679-bf1a-3b190cc36d05.png)
done.